newton/_src/utils/import_urdf.py

```
@@ -246,18 +252,80 @@ def parse_shapes(link, geoms, density, incoming_xform=None, visible=True, just_v
+        sorted_joint_ids = topological_sort(joint_edges, use_dfs=joint_ordering == "dfs")
+        sorted_joints = [joints[i] for i in sorted_joint_ids]
+    else:
+        sorted_joints = joints
+
+    if bodies_follow_joint_ordering:
+        body_order: list[str] = [sorted_joints[0]["parent"]] + [joint["child"] for joint in sorted_joints]
+        urdf_links = []
+        for body in body_order:
+            urdf_link = root.find(f"link[@name='{body}']")
+            if urdf_link is None:
+                raise ValueError(f"Link {body} not found in URDF")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)